epilogue_1_how_to_get_there_from_here.asciidoc

```diff
@@ -380,16 +414,23 @@ TODO: Context diagram, E-Commerce, Availability, Fulfilment, Event-Broker
     - or new ones, and we switch over the downstream things progressively
 
 
-
 //TODO: event capture and all that jazz
 
+// (EJ1) * "Event Capture" and "Data Capture" are used interchangeably.  Is it supposed to be
+//       CDC consistently?
+//       * Thoughworks recently put CDC on "Hold" status (https://www.thoughtworks.com/radar/techniques/puncturing-encapsulation-with-change-data-capture).  Most tools + ref material I've seen on CDC focus on row level ops which
+//       may be too low-level (e.g. Debezium), so a warning to project Domain Events here may be warranted. Mention
+//       of Outbox pattern here could be useful.
+
 === More required reading
 
 * _Monolith to Microservices_ by Sam Newman, and his original book,
    _Building Microservices_
 
 Stangler (Fig) pattern is mentioned as a favorite, also many others.
 
+// (EJ3) "Evolutionary Architecture" also?  I have yet to read it but am on-board with the general philosophy.
```

Comment:
it's a good book but i think the main thing it's pitching, "define fixture functions for all your architectural goals and measure them", is a bit orthogonal to what we're selling...   and pretty far out besides!

Reply:
Will have to take a deeper dive into it, but I was thinking that it's relevant in a microservice world, where you need to incrementally migrate your architecture, like with StranglerFig.